dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -240,6 +246,16 @@ const std::wstring NotificationsLongRunningPlatformImpl::BuildAppIdentifier(std:
     return guidStr.get();
 }
 
+void NotificationsLongRunningPlatformImpl::AddComServerClsid(std::wstring const& appId, winrt::guid const& comServerClsid)
+{
+    m_comServerClsidStorage.Values().Insert(appId, winrt::box_value(comServerClsid));
+}
+
+void NotificationsLongRunningPlatformImpl::RemoveComServerClsid(std::wstring const& appId)
+{
+    m_comServerClsidStorage.Values().Remove(appId);
```

Comment:
No locks

Reply:
Don't need this function because it is only used once. Removing.